telemetry.py

```diff
@@ -47,7 +47,7 @@ def _query(query):
 def subscription(cluster):
     """Get the cluster's subscription_labels.
     """
-    data = _query(query='topk(1, subscription_labels{{_id="{}"}} offset 3h)'.format(cluster))
+    data = _query(query=f'topk(1, max_over_time(subscription_labels{{_id="{cluster}",support!=""}}[1w]))')
```

Comment:
I don't want to have to worry about what version of Python 3 folks are running.  Can we stick to `format(...)` instead of using an f-string?

Reply:
`slackclient` requires py3 iiuc, but okay, lets stick to `.format(`

Reply:
f-strings are [new in 3.6][1], so it's not just 2 vs. 3.

[1]: https://www.python.org/dev/peps/pep-0498/